pyproject.toml

```diff
@@ -46,13 +46,13 @@ dask = {extras = ["dataframe"], version = ">= 2023.4.1"}
 importlib-resources = { version = ">= 1.3", python = "<3.9" }
 jsonschema = ">= 4.17.3"
 pyarrow = ">= 11.0.0"
+kfp = { version = ">= 1.8.19"}
 
-kfp = { version = ">= 1.8.19", optional = true }
 kubernetes = { version = ">= 18.20.0", optional = true }
 pandas = { version = ">= 1.3.5", optional = true }
 
 [tool.poetry.extras]
-pipelines = ["kfp", "kubernetes"]
+pipelines = ["kubernetes"]
```

Comment:
What's the reason for these changes? `kfp` is quite a big dependency which you don't need when installing `fondant` in a component, which is why we added it as an optional dependency to the `pipeline` extra.

Reply:
kfp is used by the pipeline.py so it is needed for `fondant compile` and `fondant run`

Reply:
Yes, so then the user needs to install `fondant[pipelines]` (`fondant[kfp]` might be better). But we don't want to install this in every component.